README.md

```diff
@@ -522,6 +522,4 @@ This approach is intended to create cohesion and keep the project sustainable.
 There are additional documents that are part of the software documentation. they are in the folder `/docs` and are listed below:
 
 - [docs/dynamic-inputs.md](docs/dynamic-inputs.md) - describe how the inputs are rendered using a custom engine and not hard-coded
-- [docs/learned-lessons.md](docs/learned-lessons.md) - lessons learned and that might help in the future
-- [docs/learned-lessons.md](docs/learned-lessons.md) - explains what are the criteria for the search results
```

Comment:
There is the file `search-results-criteria.md` which the second link should go to.
Removal of the first one is fine (although not really part of the docker change :P but we can make an exception)